src/app/components/hris/employees/employee-approvals/view-banking-approval/view-banking-approval.component.html

```diff
@@ -18,7 +18,8 @@
         <div id="approval-header" class="row p-2">
             <div class="col-2 mt-3 mb-3">
                 <div id="profile-image">
-                    <img [src]="employee?.photo == '' ? '../../../../../assets/img/default-profile-image.png' : employee?.photo" alt="">
+                    <img [src]="this.authAccessService.getAuthTokenProfilePicture() || '../../../../../../assets/img/default-profile-image.png'"
+          alt="Profile img" class="rounded-circle profile-img">>
```

Comment:
Fix: the logic here should be the same as in the others -

first use employee.photo, check if available
if employee profile photo not available, use google photo
if google photo not available use static image